share/qtcreator/sfdk/modules/30-building-maintain.json

```diff
@@ -22,10 +22,10 @@
         },
         {
             "name": "tools",
-            "trSynopsis": "[tooling|target] list [-a|--available] [--slow]\n[tooling|target] update <name>\n[tooling|target] register [--user <user> [--password <password>]] {--all|<name>}\n[tooling|target] install <name>\n[tooling|target] install-custom <name> {<URL>|<file>} [--tooling <tooling>] [--no-snapshot]\n[tooling|target] remove [--snapshots-of] <name>\n[tooling|target] package-list <name> [<pattern>...]\n[tooling|target] package-install <name> <package>...\n[tooling|target] package-remove <name> <package>...\n[tooling|target] exec <name> [<command> [<arg>...]]",
+            "trSynopsis": "[tooling|target] list [-a|--available] [--slow]\n[tooling|target] update <name>\n[tooling|target] register [--user <user> [--password <password>]] {--all|<name>}\n[tooling|target] install <name>\n[tooling|target] install-custom <name> {<URL>|<file>} [--tooling <tooling>] [--no-snapshot]\n[tooling|target] remove [--snapshots-of] <name>\n[tooling|target] package-list\n[tooling|target] package-search <name> [<pattern>...]\n[tooling|target] package-install <name> <package>...\n[tooling|target] package-remove <name> <package>...\n[tooling|target] exec <name> [<command> [<arg>...]]",
```

Comment:
I would prefer omitting the old option from the docs as we did on sdk-assistant side.

Reply:
I thought I already did that, but clearly I didn't..

Reply:
maybe now :)

Reply:
One more occurrence in the long help :)

Reply:
It's only in the sentence "The 'package-list' command is deprecated, use 'package-search' instead.", which I think is still good to have.

Reply:
We have the warning that is issued when the deprecated option is used.

Reply:
Indeed we do. I had already forgotten that.

Reply:
another try :)